### PR TITLE
Merge request for github test 

### PR DESCRIPTION
**Pull request by Surinder07**

### CODE REVIEW

File: nov23/readme.md [hunk 1]
```diff
@@ -12,4 +12,6 @@ Its a request you generate for your team mates to review your code
 before actually gets merged to main branch
 
 ## git pull
-bringin latest changes from git remote[github] to local[laptop]
\ No newline at end of file
+bringin latest changes from git remote[github] to local[laptop]
+
+## more changes on develop
```
Comment (Surinder07) [inline nov23/readme.md]: I reviewed it

Comment (ekamdeeps) [reply nov23/readme.md]: I reviewed it

**Review by ekamdeeps — APPROVED** [review]
done

File: nov23/readme.md (hunk 1 above)
Comment (RHalai-Engin) [reply nov23/readme.md]: Reviewed..

**Review by arpinder144 — APPROVED** [review]
approved

**Review by ekamdeeps — APPROVED** [review]

**Review by SukhjeetGill — APPROVED** [review]

**Review by RHalai-Engin — APPROVED** [review]
Approved

**Pull request merged**